cookiecutter.json

```diff
@@ -1,11 +1,16 @@
 {
+  "project_owner_name": "Simon Billinge",
+  "project_owner_email": "sbillinge@columbia.edu",
+  "project_owner_github_username": "sbillinge",
+  "contributors": "Billinge Group members and community contributors",
+  "license_holder": "The Trustees of Columbia University in the City of New York",
   "github_org": "diffpy",
-  "keywords": "",
-  "project_name": "diffpy.my_project",
+  "project_name": "diffpy.my-project",
   "package_dist_name": "{{ cookiecutter.project_name|replace(' ', '-')|lower }}",
   "package_dir_name": "{{ cookiecutter.project_name|replace(' ', '_')|replace('-', '_')|lower }}",
   "repo_name": "{{ cookiecutter.project_name|replace(' ', '-')|lower }}",
   "project_short_description": "Python package for doing science.",
+  "keywords": "diffraction, PDF, X-ray, neutron",
   "minimum_supported_python_version": "3.11",
   "maximum_supported_python_version": "3.13",
   "have_c_code": ["n", "y"],
```

Comment:
`project_needs_c_code_compiled` ?  or check, is ti c++?  Also, why is this is list and all the others a string? It seems to be using a different logic (showing all possible values rather than a default example value)

Reply:
https://github.com/Billingegroup/cookiecutter/pull/211#pullrequestreview-2524557433

https://github.com/Billingegroup/cookiecutter/issues/213 has been created. I will tackle this in a separate PR.